### PR TITLE
Fix Bank of Corda, Trader and Notary demos. 

### PR DESCRIPTION
**Pull request by kasiastreich**
In Notary demo, introduce deployNodes task, that deploys all three types of Notary into different directories (so there is no need to do clean, when you want to run them one after another).

### CODE REVIEW

File: samples/bank-of-corda-demo/build.gradle [hunk 1]
```diff
@@ -52,7 +52,7 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
         cordapps = []
     }
     node {
-        name "CN=BankOfCorda,O=R3,OU=Corda QA Department,L=New York,C=US"
+        name "CN=BankOfCorda,O=R3,L=New York,C=US"
```
Comment (Clintonio) [inline samples/bank-of-corda-demo/build.gradle]: What is the purpose of this change? I'm not familar with X500 names.

Comment (kasiastreich) [inline samples/bank-of-corda-demo/build.gradle]: It's because @rnicoll had some conversation with Mike what should be in X500 names and now in CoreTestUtils BOC is "CN=BankOfCorda,O=R3,L=New York,C=US", which doesn't fit the name in network map taken from this config.

File: docs/source/running-the-demos.rst [hunk 2]
```diff
@@ -123,21 +123,23 @@ You will notice that successive transactions get signed by different members of
 
 To run the Raft version of the demo from the command line in Unix:
 
-1. Run ``./gradlew samples:notary-demo:deployNodesRaft``, which will create node directories with configs under ``samples/notary-demo/build/nodes``.
-2. Run ``./samples/notary-demo/build/nodes/runnodes``, which will start the nodes in separate terminal windows/tabs.
+1. Run ``./gradlew samples:notary-demo:deployNodes``, which will create all three types of notaries nodes directories
```
Comment (rnicoll) [inline docs/source/running-the-demos.rst]: I think "notaries nodes directories" should be "notaries' node directories", although admit I get a headache trying to figure exactly where the plurals go, myself.

File: docs/source/running-the-demos.rst [hunk 3]
```diff
@@ -123,21 +123,23 @@ You will notice that successive transactions get signed by different members of
...
+2. Run ``./samples/notary-demo/build/nodes/nodesRaft/runnodes``, which will start the nodes in separate terminal windows/tabs.
    Wait until a "Node started up and registered in ..." message appears on each of the terminals
 3. Run ``./gradlew samples:notary-demo:notarise`` to make a call to the "Party" node to initiate notarisation requests
    In a few seconds you will see a message "Notarised 10 transactions" with a list of transaction ids and the signer public keys
 
 To run from the command line in Windows:
 
-1. Run ``gradlew samples:notary-demo:deployNodesRaft``, which will create node directories with configs under ``samples\notary-demo\build\nodes``.
-2. Run ``samples\notary-demo\build\nodes\runnodes``, which will start the nodes in separate terminal windows/tabs.
+1. Run ``gradlew samples:notary-demo:deployNodes``, which will create all three types of notaries nodes directories
```
Comment (rnicoll) [inline docs/source/running-the-demos.rst]: As above, "notaries nodes" to "notaries' node"

File: docs/source/running-the-demos.rst (hunk 2 above)
Comment (kasiastreich) [inline docs/source/running-the-demos.rst]: That's why I took you to that PR ;)
